.github/actions/check-pr-status/index.js

```diff
@@ -0,0 +1,44 @@
+const core = require('@actions/core');
+const { context } = require('@actions/github');
+
+const BLOCKING_LABELS = [`flag: 💥 Breaking change`, `flag: don't merge`];
+
+async function main() {
+  try {
+    const labels = context.payload.pull_request?.labels;
+
+    const hasBlockingLabel = labels.some(label => BLOCKING_LABELS.includes(label.name));
+    if (hasBlockingLabel) {
+      core.setFailed(
+        `The PR has been labelled with a blocking label (${BLOCKING_LABELS.join(', ')}).`
```

Comment:
Idea: Even if the list won't grow that much, it could maybe be more helpful to log the blocking labels that are actually set on this PR? This way you can see at a glance what's causing the issue.